processing.js

```diff
@@ -905,9 +911,9 @@ module.exports = function finalizeProcessing(Processing, options) {
    * source and bind to canvas via new Processing(canvas, sourcestring).
    * @param {CANVAS} canvas The html canvas element to bind to
    * @param {String[]} source The array of files that must be loaded
-   * @param {Function} complete A callback, called with the sketch as the argument.
+   * @param {Function} onComplete A callback, called with the sketch as the argument.
```

Comment:
let's rename this back to `complete`, to keep with the "one issue, one fix" philosophy: this is a change technically not related to getting chord/pie closing to work, so doesn't really belong in this PR